lib/net/pool.js

```diff
@@ -4494,6 +4788,21 @@ class HeaderEntry {
   }
 }
 
+class CFHeaderEntry {
+  /**
+   * Create cfheader entry.
+   * @constructor
+   */
+
+  constructor(blockHash, header, height) {
+    this.blockHash = blockHash;
+    this.header = header;
+    this.height = height;
```

Comment:
rename them to filter header and filter height.
also add a field for the filterType

Reply:
I think we should keep height the same as it is by logic the height of a chain. We could rename header to filterHeader.